WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -131,6 +133,8 @@ public void Start(TimeSpan requestInterval, TimeSpan feeQueryRequestInterval, in
 
 			Task.Run(async () =>
 			{
+				Logger.LogTrace("> Starting Wasabi synchronizer thread starts.");
```

Comment:
Did you mean to say `> Wasabi synchronizer thread starts.`?

Reply:
Yes!